fcrepo-configs/src/main/java/org/fcrepo/config/FedoraPropsConfig.java

```diff
@@ -35,20 +35,60 @@
  * @since 6.0.0
  */
 @Configuration
-public class FedoraPropsConfig {
+public class FedoraPropsConfig extends BasePropsConfig {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(FedoraPropsConfig.class);
 
+    public static final String FCREPO_CONFIG_FILE = "fcrepo.config.file";
     public static final String FCREPO_HOME = "fcrepo.home";
+    public static final String FCREPO_JMS_HOST = "fcrepo.jms.host";
+    public static final String FCREPO_DYNAMIC_JMS_PORT = "fcrepo.dynamic.jms.port";
+    public static final String FCREPO_DYNAMIC_STOMP_PORT = "fcrepo.dynamic.stomp.port";
+    public static final String FCREPO_ACTIVEMQ_CONFIGURATION = "fcrepo.activemq.configuration";
+    public static final String FCREPO_NAMESPACE_REGISTRY = "fcrepo.namespace.registry";
+    public static final String FCREPO_EXTERNAL_CONTENT_ALLOWED = "fcrepo.external.content.allowed";
+    private static final String FCREPO_ACTIVEMQ_DIRECTORY = "fcrepo.activemq.directory";
+
+    private static final String DATA_DIR_DEFAULT_VALUE = "data";
+    private static final String ACTIVE_MQ_DIR_DEFAULT_VALUE = "ActiveMQ/kahadb";
 
-    private static final String DATA_DIR = "data";
 
     @Value("${" + FCREPO_HOME + ":fcrepo-home}")
     private Path fedoraHome;
 
-    @Value("#{fedoraPropsConfig.fedoraHome.resolve('" + DATA_DIR + "')}")
+    @Value("#{fedoraPropsConfig.fedoraHome.resolve('" + DATA_DIR_DEFAULT_VALUE + "')}")
     private Path fedoraData;
 
+    @Value("${" + FCREPO_JMS_HOST + ":localhost}")
+    private String jmsHost;
+
+    @Value("${" + FCREPO_DYNAMIC_JMS_PORT + ":61616}")
+    private String jmsPort;
+
+    @Value("${" + FCREPO_DYNAMIC_STOMP_PORT + ":61613}")
+    private String stompPort;
+
+
+    @Value("${" + FCREPO_ACTIVEMQ_CONFIGURATION + ":classpath:/config/activemq.xml}")
+    private String activeMQConfiguration;
+
+    @Value("${" + FCREPO_ACTIVEMQ_DIRECTORY + ":#{fedoraPropsConfig.fedoraData.resolve('" +
+            ACTIVE_MQ_DIR_DEFAULT_VALUE + "').toAbsolutePath().toString()}}")
+    private String activeMqDirectory;
+
+    @Value("${" + FCREPO_NAMESPACE_REGISTRY + ":classpath:/namespaces.yml}")
+    private String namespaceRegistry;
+
+    @Value("${" + FCREPO_EXTERNAL_CONTENT_ALLOWED + ":#{null}}")
+    private String externalContentAllowed;
+
+    @Value("${fcrepo.session.timeout:180000}")
```

Comment:
Is there a reason why the `fcrepo.session.timeout` has not be extracted into a static constant like the ones above?
Same question for `fcrepo.velocity.runtime.log` below.

Reply:
It's just that I don't think they're referenced anywhere else.   I'll make them consistent.